platform/metrics/platform-metrics-reporting/pom.xml

```diff
@@ -28,25 +28,21 @@
             <groupId>org.rrd4j</groupId>
             <artifactId>rrd4j</artifactId>
         </dependency>
+        <!-- DDF-3064 poi versions need to be updated to ${poi.version}-->
         <dependency>
             <groupId>org.apache.poi</groupId>
             <artifactId>poi</artifactId>
-            <version>3.9</version>
-        </dependency>
-        <dependency>
-            <groupId>org.apache.poi</groupId>
-            <artifactId>poi-contrib</artifactId>
-            <version>3.7-beta3</version>
+            <version>3.12</version>
```

Comment:
❓ Can we define the `${poi.version}` in this pom instead of repeating the version?